### PR TITLE
Adds a schedule cve scanning action

### PR DESCRIPTION
**Pull request by mmochan**
#### What does this PR do?
Adds a scheduled GitHub action to scan for CVE vulnerabilities, which if detected will alert in slack.

#### Type of change
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: .github/workflows/scan.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout git
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get unstable git tag
+        run: >
+          echo ::set-env name=UNSTABLE_TAG::$(git describe --tags)
```
Comment (Kirill888) [inline .github/workflows/scan.yml]: `echo ::set-env ...` syntax recently got deprecated and will be removed

File: .github/workflows/scan.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+      - name: Log the unstable tag
+        run: echo $UNSTABLE_TAG
+
+      - name: Build unstable + latest Docker image tag
```
Comment (Kirill888) [inline .github/workflows/scan.yml]: I think we should pull from hub rather than build fresh.

Comment (Kirill888) [reply .github/workflows/scan.yml]: Also there is not one but 3 image tags, yes it's not ideal...

Comment (mmochan) [reply .github/workflows/scan.yml]: Did you want me to scan all 3 images?

Comment (Kirill888) [reply .github/workflows/scan.yml]: I guess so, currently we only run `:runner`  derived images in k8s, but I think we should start running builder/wheels in k8s as well to compile our custom wheels

Comment (mmochan) [reply .github/workflows/scan.yml]: I've added all 3 images, pulling from Docker Hub.

**Review by Kirill888 — APPROVED** [review]
looks fine, but would be nice to see it actually run before merging

**Pull request merged**